src/ceph-create-keys

```diff
@@ -84,12 +84,12 @@ def get_key(cluster, mon_id):
     pathdir = os.path.dirname(path)
     if not os.path.exists(pathdir):
         os.makedirs(pathdir)
-        os.chmod(pathdir, 0o770)
+        os.chmod(pathdir, 0o770 | 0770)
```

Comment:
@thebinarymutant i think `0o770` is good enough? why would you OR it with `0770`?